airbyte-integrations/connectors/source-salesforce/Dockerfile

```diff
@@ -13,5 +13,5 @@ RUN pip install .
 
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=1.0.9
+LABEL io.airbyte.version=1.1.0
```

Comment:
+1. this doesn't have to be a minor release

Reply:
Oh, my bad. How do I fix this? Pushing a commit gives me the following error:

error: failed to push some refs to 'github.com:airbytehq/airbyte.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

Reply:
The obvious answer seems to be `git pull` then `git push` but want to make sure that's correct

Reply:
Okay, I changed the version in the dockerfile. How do I update the source_specs and source_definitions files?

Reply:
I think I fixed it. Can someone verify and tell me what to do next?